test/optimizer/join_reorder_optimizer.test

```diff
@@ -33,35 +33,80 @@ statement ok
 PRAGMA debug_force_no_cross_product=true
 
 statement ok
-EXPLAIN 
-SELECT 
-  COUNT(*) 
-FROM 
-  t1, t2, t3, t4 
-WHERE 
-  t1.c1 = t2.c1 AND 
-  t2.c2 = t3.c2 AND 
+EXPLAIN
+SELECT
+  COUNT(*)
+FROM
+  t1, t2, t3, t4
+WHERE
+  t1.c1 = t2.c1 AND
+  t2.c2 = t3.c2 AND
   t3.c3 = t4.c3
 
 statement ok
-EXPLAIN 
-SELECT 
-  COUNT(*) 
-FROM 
-  t1, t2, t3, t4 
-WHERE 
-  t1.c1 = t2.c1 AND 
-  t2.c2 = t3.c2 AND 
-  t3.c3 = t4.c3 AND 
+EXPLAIN
+SELECT
+  COUNT(*)
+FROM
+  t1, t2, t3, t4
+WHERE
+  t1.c1 = t2.c1 AND
+  t2.c2 = t3.c2 AND
+  t3.c3 = t4.c3 AND
   t4.c4 = t1.c4
 
 statement ok
-EXPLAIN 
-SELECT 
-  COUNT(*) 
-FROM 
-  t1, t2, t3, t4 
-WHERE 
-  t1.c1 = t2.c1 AND 
-  t2.c2 = t3.c2 AND 
+EXPLAIN
+SELECT
+  COUNT(*)
+FROM
+  t1, t2, t3, t4
+WHERE
+  t1.c1 = t2.c1 AND
+  t2.c2 = t3.c2 AND
   t1.c1 + t2.c2 + t3.c3= 3 * t4.c4
+
+statement ok
+PRAGMA debug_force_no_cross_product=false
+
+statement ok
```

Comment:
Do you think we should verify the query results? Maybe we can turn the `exists` into a `*`?